com.avaloq.tools.ddk.xtext.ui.test/pom.xml

```diff
@@ -0,0 +1,66 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>ddk-parent</artifactId>
+    <groupId>com.avaloq.tools.ddk</groupId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../ddk-parent</relativePath>
+  </parent>
+
+  <artifactId>com.avaloq.tools.ddk.xtext.ui.test</artifactId>
+  <packaging>eclipse-test-plugin</packaging>
+
+  <build>

```

Comment:
This whole section seems awfully suspicious to me. Do we need it? Should it not come from the parent pom?

Reply:
Removed it.